pep-3156.txt

```diff
@@ -361,7 +373,8 @@ stopping and closing.  (However, a partially-conforming event loop is
 still better than nothing. :-)
 
 - Starting, stopping and closing: ``run_forever()``,
-  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``.
+  ``run_until_complete()``, ``stop()``, ``is_running()``, ``close()``,
```

Comment:
Yep, I agree with @gvanrossum 
Let's keep these methods as part of public API -- they are almost useless for casual user but describes the current loop very well.